src/libraries/MathLib.sol

```diff
@@ -10,11 +10,13 @@ int256 constant WAD_INT = int256(WAD);
 /// @custom:contact security@morpho.org
 /// @notice Library to manage fixed-point arithmetic on signed integers.
 library MathLib {
-    function wMulDown(int256 a, int256 b) internal pure returns (int256) {
+    /// @dev Returns the multiplication of `a` by `b` (in WAD) rounded towards 0.
+    function wMulTo0(int256 a, int256 b) internal pure returns (int256) {
         return a * b / WAD_INT;
```

Comment:
Nitpick: follow the same style and names used by Morpho Blue `MathLib` contract

- `a * b / WAD_INT;` → `(a * b) / WAD_INT;`
- `a` → `x` and `b` → `y`
- update the natspec documentation reflecting the changes (always following the MB natspec)

Reply:
It's strange because this is a change that is automagically suggested and applied by the formatter (wrapping terms in parentheses)

Reply:
indeed would have been better but we won't change it since it's not that important